internal/status.go

```diff
@@ -0,0 +1,152 @@
+package internal
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/dustin/go-humanize"
+)
+
+type StatusLine struct {
+	resources []Resource
```

Comment:
It's preferable to have a *pointer* to a list of resources, to avoid duplicating data.